data/human/free worlds side plots.txt

```diff
@@ -11,6 +11,85 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+
+
+mission "FW Convoy to Zug"
+	name "Convoy to Zug"
+	description "Escort these Haulers to Zug for an important project. Expect pirate raids."
+	source
+		government "Free Worlds"
+		near Zubeneschamali 2 4
+	destination "Zug"
+	to offer
+		has "FW Katya 7C: done"
+		not "main story completed"
+	on offer
+		conversation
+			`While returning from a walk around the spaceport town of <origin>, you see the quartermaster of the spaceport inspecting the <ship>. He notices your arrival, and comes up to greet you.`
+			`	"You're Captain <first> <last>, yes? We're looking for a good ship that can escort these freighters to <planet>." He points towards five Haulers of varying sizes in Free Worlds colors, some of which have considerable hull damage and are being repaired on the landing pads opposite to your ship.`
+			`	"Their last escort was just taken out by pirates, and we need someone who is experienced enough to fend off or evade them. You must be well known here, because many of our militia officers here recommended you. This is for quite a significant project on <planet>, so you will be paid <payment>."`
```

Comment:
```suggestion
			`	"Their last escort was just taken out by pirates, and we need someone who is experienced enough to fend off or evade them. You must have a reputation, since some of our militia captains recognized you. This is for quite a significant project on <planet>, so you will be paid <payment>."`
```

Slight wording tweak to adjust for the fact that the player is getting this based more on combat ability and not forcibly by being already involved with the FW.